CHANGELOG.md

```diff
@@ -107,6 +110,32 @@ Main (unreleased)
 - RPM artifacts in Alloy GitHub releases are no longer signed.
   The artifacts on the `https://rpm.grafana.com` repository used by the `yum` package manager will continue to be signed. (@ptodev)
 
+- Upgrade `otelcol` components from OpenTelemetry v0.122.0 to v0.125.0 (@ptodev):
+  - [`pkg/ottl`] Enhance the Decode OTTL function to support all flavors of Base64.
+  - [`otelcol.processor.resourcedetection`] Adding the `os.version` resource attribute to system processor.
+  - [`otelcol.auth.bearer`] Allow the header name to be customized.
+  - [`otelcol.exporter.awss3`] Add a new `sending_queue` feature.
+  - [`otelcol.exporter.awss3`] Add a new `timeout` argument.
+  - [`otelcol.exporter.awss3`] Add a new `resource_attrs_to_s3` configuration block.
+  - [`otelcol.exporter.awss3`] Fixes an issue where the AWS S3 Exporter was forcing an ACL to be set, leading to unexpected behavior in S3 bucket permissions.
+  - [`otelcol.connector.spanmetrics`] A new `include_instrumentation_scope` configuration argument.
+  - [`otelcol.connector.spanmetrics`] Initialise new `calls_total` metrics at 0.
+  - [`otelcol.connector.spanmetrics`] A new `aggregation_cardinality_limit` configuration argument 
+    to limit the number of unique combinations of dimensions that will be tracked for metrics aggregation.
+  - [`otelcol.connector.spanmetrics`] Deprecate the unused argument `dimensions_cache_size`.
+  - [`otelcol.connector.spanmetrics`] Moving the start timestamp (and last seen timestamp) from the resourceMetrics level to the individual metrics level. 
+  - [`otelcol.processor.k8sattributes`] Add option to configure automatic resource attributes - with annotation prefix.
+    Implements [Specify resource attributes using Kubernetes annotations](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/k8s-attributes.md#specify-resource-attributes-using-kubernetes-annotations).
+    This will ensure that each metric has its own accurate start and last seen timestamps, regardless of its relationship to other spans.
```

Comment:
```suggestion
  - [`otelcol.connector.spanmetrics`] Moving the start timestamp (and last seen timestamp) from the resourceMetrics level to the individual metrics level. 
    This will ensure that each metric has its own accurate start and last seen timestamps, regardless of its relationship to other spans.
  - [`otelcol.processor.k8sattributes`] Add option to configure automatic resource attributes - with annotation prefix.
    Implements [Specify resource attributes using Kubernetes annotations](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/k8s-attributes.md#specify-resource-attributes-using-kubernetes-annotations).
```